### PR TITLE
Add Webots R2021b ROS 2 blog section

### PR DESCRIPTION
**Pull request by lukicdarkoo**
https://www.cyberbotics.com/doc/blog/Webots-2021-b-release?version=documentation-blog-ros2

### CODE REVIEW

File: docs/blog/Webots-2021-b-release.md [hunk 1]
```diff
@@ -112,7 +112,20 @@ The [SICK S300](../guide/lidar-sensors.md#sick-s300) lidar model is now included
...
+- [`webots_ros2_driver`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_driver): A new Webots ROS 2 interface core designed for performance, extensibility with C++ plugins, and an easy configuration through URDF files.
+- [`webots_ros2_control`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_control): A `webots_ros2` plugin that integrates the [ros2_control](https://ros-controls.github.io/control.ros.org/) framework. The plugin is featured in the [webots_ros2_turtlebot](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_turtlebot) package.
+
+We haven't forgotten about the old good ROS 1. It now integrates the [ros_control](http://wiki.ros.org/ros_control) framework and you don't have to activate sensors with services anymore.
+These improvements make Webots much simpler to integrate into ROS 1 applications (see [webots_ros#60](https://github.com/cyberbotics/webots_ros/pull/60/files)).
+We also included a Tiago example that features SLAM and MoveIt integration.
```
Comment (lukicdarkoo) [inline docs/blog/Webots-2021-b-release.md:128]: @stefaniapedrazzi, maybe we can include here the OpenDR video?

Comment (stefaniapedrazzi) [reply docs/blog/Webots-2021-b-release.md:128]: Yes, but usually we only link edited videos with titles, etc.
I think in the blog we should try to be concise and videos should be short and immediately show the functionality.

So OpenDR video as-is is not probably ready to be linked.
But we could extract part of it (for example the SLAM) and included it as animated GIF.

**Review by omichel — APPROVED** [review]

File: docs/blog/Webots-2021-b-release.md [hunk 2]
```diff
@@ -112,7 +112,20 @@ The [SICK S300](../guide/lidar-sensors.md#sick-s300) lidar model is now included
 
 ## ROS 2 & Webots
 
-TODO
+We are continuing to develop and improve the support for ROS 2 in Webots with now 16 Webots packages for ROS 2. This version of Webots is fully compatible with the two latest versions of ROS 2 and the rolling release:
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-b-release.md:115]: Once sentence per line
```suggestion
We are continuing to develop and improve the support for ROS 2 in Webots with now 16 Webots packages for ROS 2.
This version of Webots is fully compatible with the two latest versions of ROS 2 and the rolling release:
```

File: docs/blog/Webots-2021-b-release.md [hunk 3]
```diff
@@ -112,7 +112,20 @@ The [SICK S300](../guide/lidar-sensors.md#sick-s300) lidar model is now included
...
+- [`webots_ros2_mavic`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_mavic): The package demonstrates a simulation of drones with Webots and ROS 2. The DJI' Mavic 2 PRO drone uses a simple PD controller to follow the given [geometry\_msgs/Twist](https://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Twist.html) commands.
+- [`webots_ros2_driver`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_driver): A new Webots ROS 2 interface core designed for performance, extensibility with C++ plugins, and an easy configuration through URDF files.
+- [`webots_ros2_control`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_control): A `webots_ros2` plugin that integrates the [ros2\_control](https://ros-controls.github.io/control.ros.org/) framework. The plugin is featured in the [webots\_ros2\_turtlebot](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_turtlebot) package.
+
+We haven't forgotten about the old good ROS 1. It now integrates the [ros\_control](http://wiki.ros.org/ros_control) framework and you don't have to activate sensors with services anymore.
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-b-release.md:126]: One sentence per line
```suggestion
We haven't forgotten about the old good ROS 1.
It now integrates the [ros\_control](http://wiki.ros.org/ros_control) framework and you don't have to activate sensors with services anymore.
```

File: docs/blog/Webots-2021-b-release.md [hunk 4]
```diff
@@ -112,7 +112,20 @@ The [SICK S300](../guide/lidar-sensors.md#sick-s300) lidar model is now included
...
+These improvements make Webots much simpler to integrate into ROS 1 applications (see [webots\_ros#60](https://github.com/cyberbotics/webots_ros/pull/60/files)).
```
Comment (stefaniapedrazzi) [inline docs/blog/Webots-2021-b-release.md:127]: I'm not sure it is really good to link to PR in the blog.
Personally I would avoid it and remove the link.

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]
Thank you.
I just have some minor comments.

File: docs/blog/Webots-2021-b-release.md (hunk 4 above)
Comment (lukicdarkoo) [reply docs/blog/Webots-2021-b-release.md:129]: I like that PR, it extends functionalities while deleting a lot of code :)

Comment (stefaniapedrazzi) [reply docs/blog/Webots-2021-b-release.md:129]: Ok. So let's keep it.
But still if it is a good PR, I continue to think that the blog it is not the right place where to provide all the details about implementations and changes and how to port code to the new system.
We have the ChangeLog where to provide more details.

**Review by stefaniapedrazzi — APPROVED** [review]

**Pull request merged**